docs/content/doc/advanced/config-cheat-sheet.en-us.md

```diff
@@ -435,7 +433,7 @@ set name for unique queues. Individual queues will default to
 - `ADAPTER`: **memory**: Cache engine adapter, either `memory`, `redis`, or `memcache`.
 - `INTERVAL`: **60**: Garbage Collection interval (sec), for memory cache only.
 - `HOST`: **\<empty\>**: Connection string for `redis` and `memcache`.
-   - Redis: `network=tcp,addr=127.0.0.1:6379,password=macaron,db=0,pool_size=100,idle_timeout=180`
+   - Redis: `redis://macaron@127.0.0.1:6379/0?pool_size=100&idle_timeout=180s`
```

Comment:
`redis://:macaron@127.0.0.1:6379/0?pool_size=100&idle_timeout=180s`.

Reply:
It doesn't need the preceding colon. This bit comes from compatibility with the lettuce.io's variant:

https://github.com/lettuce-io/lettuce-core/wiki/Redis-URI-and-connection-details

Usernames are relatively new in redis so the uri is designed that if you have a x@host x is the password and only sets the username if you have x:y@host.

(This is still compatible with the perl version:

https://metacpan.org/pod/URI::redis
)

Reply:
Since we are making the connection uniform, I think we should restrict the configurations. The configuration should not be redis specific.

Reply:
done